microsoft-teams/installation.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Microsoft Teams'
+---
+
+**Never miss another message from a customer in Microsoft Teams**
+
+<Warning>
+    Microsoft Teams integration is currently in beta. The integration won't currently be available for your workspace until we've enabled it for you.
+
+    If you need any assistance or have feedback on the installation process please [reach out to us](mailto:help@plain.com)!
+
+</Warning>
+
+Our Microsoft Teams integration lets you sync messages from selected Teams channels to Plain and respond directly to customers from the platform.
+
+### Prerequisites
+
+For the integration you'll need:
+
+- A Teams account with at least one seat
+- The installation of the app for Teams can only be performed by an admin as you may need to change settings to:
+  - [Allow external apps to be installed](https://learn.microsoft.com/en-us/microsoft-365/admin/manage/external-app-setup?view=o365-worldwide)
+  - [Allow external users to be added as guests to be invited to Teams](https://learn.microsoft.com/en-us/microsoft-365/solutions/collaborate-as-team)
+
+## Installation
+
+<Warning>
+  🚧 As we're still building the integration, some features may not work as expected or be missing,
+  and some functionality may change. There is a list below of known issues and limitations!
+</Warning>
+
+<Steps>
+    <Step title="Connect your Plain workspace to Teams">
+        - In Plain, navigate to **`Settings`** > **`Microsoft Teams`**.
+        - Press **`Connect to Teams`**.
+        - Follow the instructions in the pop-up window to sign in to your Microsoft account, make sure you've selected the correct
+        Microsoft account if you have multiple.
+        - Once the authorization is complete, you can download the Plain app for Teams (used in the next step).
+        - Optional: Authorize replies from your Teams profile to respond directly from Plain as yourself.
+    </Step>
+
+    <Step title="Add the Plain app to Teams">
+        <Warning>
+            The method of installation here will depend on how Teams is set up for your organisation. If you are having trouble
+            with setting this up, please get in touch so we can help!
+        </Warning>
+
+        #### Option 1 - Install via Teams App
+        - Navigate to the Apps tab
+        - Click Manage your apps
+        - Click Upload an app
+        - As a Teams admin, you should be able to 'Upload an app to your organization's app catalogue' and upload the .zip file you downloaded in the previous step
+
+        #### Option 2 - Install via the [Teams Admin Dashboard](https://admin.teams.microsoft.com/dashboard)
+        - In the left sidebar, navigate to Teams apps -> Manage Apps
+        - Under Actions, click on 'Upload new app'
+        - Upload the .zip file you downloaded in the previous step
```

Comment:
Some consistency in the section titles would be good. Eg, quoting them like you do with `'Upload new app'`. Either quote or emphasis (bold) would work. The latter makes it easier to read/follow IMO